roles/create_aws_backup_storage/tasks/main.yml

```diff
@@ -1,16 +1,25 @@
+- name: Get defaults
+  include_vars:
+    file: "{{ playbook_dir }}/config/defaults.yml"
+
 - name: Get AWS account information
   aws_caller_facts:
   register: caller_facts
 
 - name: Retrieve AWS account identifier (ARN) for use in resource tags
   set_fact:
     aws_arn: "{{ caller_facts.arn }}"
-    aws_resource_name: "velero-{{ caller_facts.arn | hash('md5') }}"
+    arn_region: "{{ caller_facts.arn }}-{{ awsRegion }}"
+
+# Set this separately to explicitly mention we are setting md5 hash to include uniqueness across regions
+- set_fact:
+    aws_resource_name: "velero-{{ arn_region | hash('md5') }}"
 
 - name: Create S3 bucket for backup storage of OpenShift resource definitions
   s3_bucket:
     state: present
     name: "{{ aws_resource_name }}"
+    region: "{{ awsRegion }}"
```

Comment:
I believe ansible convention is to use `snake_case` for variable names

Reply:
Yeah I screwed this up across multiple places... since I'm already using awsRegion elsewhere let me update to snake_case in a separate PR. But thank you for pointing that out.

Reply:
Just kidding... I added it to this PR. Lot of changes now :)